generator/explores/glean_ping_explore.py

```diff
@@ -10,9 +11,43 @@ class GleanPingExplore(PingExplore):
 
     type: str = "glean_ping_explore"
 
+    def _to_lookml(self) -> dict:
+        """Generate LookML to represent this explore."""
+        views_lookml = self.get_view_lookml(self.views["base_view"])
+
+        # The first view, by convention, is always the base view with the
+        # majority of the dimensions from the top level.
+        base = views_lookml["views"][0]
+        base_name = base["name"]
+
+        joins = []
+        for view in views_lookml["views"][1:]:
+            view_name = view["name"]
+            metric = "__".join(view["name"].split("__")[1:])
+            join = {
+                "name": view["name"],
+                "relationship": "one_to_many",
+                "sql": f"CROSS JOIN UNNEST(${{{base_name}.{metric}}}) AS {view_name}",
```

Comment:
Looking over the explore, we should probably use a `LEFT JOIN UNNEST`. That would allow users to get client counts for many different counters without affecting the underlying table (i.e. all clients would still be present).

Reply:
Is left join unnest possible without having a primary key?

Reply:
I played with this for a bit; it seems like choosing a LEFT JOIN will have many of the same constraints as the CROSS JOIN, except that it also includes the null value as a field. Should each view hold a reference to the client id, so we can join on it as the primary key?

Reply:
I set the primary key to the document id, used a left join. With this, it's possible to join between the different unnested views. However, there are issues using sums from the base table with the unnested one, results in:

```
A LookML model issue occurred.
Non-unique value/primary key (or sql_distinct_key), value overflow or collision when computing sum.
```

The cross join did not have the ability to compare labeled measures against each other. The left join alone had issues computing the sum of aggregates for a single unnested table.